geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/command/ExecuteFunction.java

```diff
@@ -113,7 +113,7 @@ public void cmdExecute(final Message clientMessage, final ServerConnection serve
       FunctionStats stats = FunctionStats.getFunctionStats(functionObject.getId());
 
       // check if the caller is authorized to do this operation on server
-      functionObject.getRequiredPermissions(null).forEach(securityService::authorize);
+      functionObject.getRequiredPermissions(null,args).forEach(securityService::authorize);
```

Comment:
I'm surprised spotless didn't complain, but please use a space after every comma in all of the changes in this PR.